core/src/test/java/org/apache/calcite/test/JdbcAdapterTest.java

```diff
@@ -135,6 +135,14 @@ class JdbcAdapterTest {
             + "WHERE \"product_id\" = 1");
   }
 
+  @Test void testFilterUnionIncludingWithPlan() {
+    CalciteAssert.model(FoodmartSchema.FOODMART_MODEL)
+        .query("  ( with a as (select * from \"sales_fact_1997\")  select * from a)\n"
```

Comment:
It is better to re-beautify the format, e.g. remove inappropriate spaces

Reply:
Done